src/lunchselector/core.clj

```diff
@@ -1,7 +1,42 @@
 (ns lunchselector.core
-  (:gen-class))
+  (:require [ring.util.response :as res]
+            [bidi.ring :refer (make-handler)]
+            [ring.middleware.session :refer [wrap-session]]
+            [lunchselector.fetch :refer [get-categories]]
+            [cheshire.core :refer :all]))
```

Comment:
Try to avoid `:refer :all` where possible. This makes it difficult to track definitions for functions, and thereby refactor.